changes.d/5809.feat.d

```diff
@@ -0,0 +1,2 @@
+The workflow-state command and xtrigger now take univeral IDs instead of separate
+arguments for cycle point, task name, etc., and are flow-aware. 
```

Comment:
```suggestion
The workflow-state command and xtrigger are now flow-aware and take universal IDs instead of separate arguments for cycle point, task name, etc. (which are still supported, but deprecated). 
```